addons/website/static/src/snippets/s_countdown/000.js

```diff
@@ -175,16 +175,26 @@ const CountdownWidget = publicWidget.Widget.extend({
      * @private
      */
     _render: function () {
+
+        // Todo: This should be remove once JWEditor bug fixed.
```

Comment:
Hello 
Is this comment and/or code still needed since the revert of the JW Editor? https://github.com/odoo/odoo/commit/e5572c317a7775a58675ed73efc89b5c9f6c0c39

Thanks

Reply:
Hi @sbuhl 
revert changes is adopted.